tools/src/main/java/org/dashj/platform/tools/WalletTool.java

```diff
@@ -345,6 +359,16 @@ public static void main(String[] args) throws Exception {
         parser.accepts("dump-lookahead");
         OptionSpec<String> refundFlag = parser.accepts("refund-to").withRequiredArg();
         OptionSpec<String> txHashFlag = parser.accepts("txhash").withRequiredArg();
+
+        // username
+        OptionSpec<String> usernameFlag = parser.accepts("username").withRequiredArg();
+
+        // coinjoin
+        mixAmountFlag = parser.accepts("amount").withRequiredArg();
+        sessionsFlag = parser.accepts("sessions").withRequiredArg().ofType(Integer.class);
+        roundsFlag = parser.accepts("rounds").withRequiredArg().ofType(Integer.class);
+        multiSessionFlag = parser.accepts("multi-session").withOptionalArg().ofType(Boolean.class);
+
```

Comment:
This WalletTool is copied from DashJ and it differs in that it has some DashPay functionality.  Some of that functionality is for testing only and not useful for a user.

In this PR, the user can finally specify a username.  Before it was randomly generated.